tests/queries/0_stateless/02340_parts_refcnt_mergetree.sh

```diff
@@ -15,9 +15,10 @@ function check_refcnt_for_table()
     query_id="$table-$(random_str 10)"
 
     # Notes:
-    # - query may sleep 0.1*(200/4)=5 seconds, it is enough to check system.parts
+    # - query may sleep 1*(200/4)=50 seconds maximum, it is enough to check system.parts
```

Comment:
probably then `-- Tags: no-fasttest`?

Reply:
I don't think that it is necessary, 50 seconds is a maximum timeout, usually it will take less.
And even 50 seconds is OK, but the problem here is that there are two tables that in total can exceed default timeout of 60 seconds.
Anyway this is not related to `no-fasttest` but more to `long` tag.

Reply:
ok, but then what about `long`?

Reply:
No it does not fail, so I don't see the reason for adding it, but if you want, let me know - will do, I don't have strong objections here (though I would prefer to keep less tests with long tag).

Reply:
yes, on the one side we want to found problems earlier with fast tests, on the flip side we really don't want to have flaky tests. I would not expect parts refcounting to be often broken by pr-s, and hence I'd prefer to mark this test as `long` for the sake of its stability.

Reply:
Ok. Added.